NaturalSolutions/geocam
# GitHub Actions (test/build/documentation/code style)

Conversion of GitLab actions:
- Build documentation
- Test API
- Build API images and push to GitHub Container Registry
- Check code formatting

Deploy action not converted, perhaps could stay on GitLab.
Probably some optimisation could be done for build conditions (e.g. only building API when changes are made to the API path, separating frontend/backend into their own workflows) but that can wait until after this PR has been approved (or not).
